docs/modules/ROOT/pages/cli_reference.adoc

```diff
@@ -70,6 +70,18 @@ Starts the operator in the cluster.
 
 include::cmd:ike[flags='serve --help --help-format=adoc']
 
+[#ike-create]
+=== `ike create`
+
+Creates or joins existing development session in the cluster. Allows to re-route cluster network traffic to your local process.
+
+The `create` command require an image reference to a repository reachable from within the cluster and will prepare a session with the given image.
```

Comment:
```suggestion
The `create` command requires an image reference to a repository which is reachable from within the cluster. It will prepare a session with the given image.
```

Reply:
fixed